cli/deploy-custom-container-triton.sh

```diff
@@ -51,7 +62,7 @@ SCORING_URL=$(az ml online-endpoint show -n $ENDPOINT_NAME --query scoring_uri -
 echo "Scoring url is $SCORING_URL"
 
 # <test_online_endpoint>
-python endpoints/online/triton/single-model/triton_densenet_scoring.py --base_url $SCORING_URL --token $KEY
+python endpoints/online/triton/single-model/triton_densenet_scoring.py --base_url $SCORING_URL --token $KEY --image_url endpoints/online/triton/single-model/peacock.jpg
```

Comment:
Perhaps move the image to a folder "data" or "image" to keep only text files in the single-model folder